librz/core/cmd/cmd_open.c

```diff
@@ -563,7 +563,8 @@ static void rz_core_cmd_omt(RzCore *core, const char *arg) {
 	rz_table_free(t);
 }
 
-static void cmd_open_map(RzCore *core, const char *input) {
+RZ_IPI int rz_om_oldinput(void *data, const char *input) {
```

Comment:
What about converting the rest of the `om` commands and removing this function completely?

Reply:
I will, but I prefer to do step by step as i have time instead of blocking on a huge PR that takes so long to finish and review. This way allows me to convert single om commands as i move forward.